include/LightGBM/feature_group.h

```diff
@@ -181,6 +185,8 @@ class FeatureGroup {
   std::unique_ptr<Bin> bin_data_;
   /*! \brief True if this feature is sparse */
   bool is_sparse_;
+  /*! \brief Threshold for treating a feature as a sparse feature */
+  double sparse_threshold_;
```

Comment:
Thanks for the clarification! I have moved `sparse_threshold` to the `Dataset` class in commit `cb35a028`